packages/rocketchat-statistics/server/lib/SAUMonitor.js

```diff
@@ -0,0 +1,410 @@
+import { Meteor } from 'meteor/meteor';
+import { Accounts } from 'meteor/accounts-base';
+import UAParser from 'ua-parser-js';
+import { UAParserMobile } from './UAParserMobile';
+import { Sessions } from 'meteor/rocketchat:models';
+import { Logger } from 'meteor/rocketchat:logger';
+import { SyncedCron } from 'meteor/littledata:synced-cron';
+
+const getDateObj = (dateTime) => {
+	if (!dateTime) {
+		dateTime = new Date();
+	}
+
+	return { day: dateTime.getDate(), month: dateTime.getMonth() + 1, year: dateTime.getFullYear() };
+};
+
+const isSameDateObj = (oldest, newest) => {
+	const oldDate = new Date(oldest.year, oldest.month - 1, oldest.day);
+	const newDate = new Date(newest.year, newest.month - 1, newest.day);
+
+	return oldDate.valueOf() === newDate.valueOf();
+};
+
+const logger = new Logger('SAUMonitor');
+
+/**
+ * Server Session Monitor for SAU(Simultaneously Active Users) based on Meteor server sessions
+ */
+export class SAUMonitorClass {
+	constructor() {
+		this._serviceName = 'SAUMonitor';
+		this._started = false;
```

Comment:
any reason to have this `started` state? I haven't found anything stopping it. and from what I saw there is nothing trying to start it multiple times as well.

Reply:
This is just a precaution I guess